src/ConnectedMode/UI/ManageConnections/ManageConnectionsDialog.xaml

```diff
@@ -96,7 +96,7 @@
                                                        Margin="10,0" />
                                 </Button>
                                 <Button Grid.Column="1" Style="{StaticResource IconButtonStyle}" ToolTip="{x:Static res:UiResources.RemoveConnectionToolTip}" Click="RemoveConnectionButton_OnClick">
-                                    <imaging:CrispImage Width="20" Height="20" VerticalAlignment="Center" Cursor="Hand" Moniker="{x:Static vsimagecatalog:KnownMonikers.Close}"/>
+                                    <imaging:CrispImage VerticalAlignment="Center" Cursor="Hand" Moniker="{x:Static vsimagecatalog:KnownMonikers.Remove}"/>
```

Comment:
Although it's more visible on dark background than the dark icon we had before, I find it unclear/confusing. Its look doesn't resemble a button, probably because it's a dash at the far right side of the window without padding.

![image](https://github.com/user-attachments/assets/7f5f4c34-9f98-4d72-a913-4276e68880db)

Maybe adding back the padding and the background colors will help.

![image](https://github.com/user-attachments/assets/fac744d1-d829-4662-9d1c-daf3823fdebc)

It requires some designer's input here.

Reply:
I added the "delete" image from the Visual Studio 2022 Image Library

Reply:
Also the "check" one as it was also not visible